contributions-chapter/chapter.md

```diff
@@ -1,15 +1,98 @@
 # Contributions for DESOSA 2017
 
-This chapter outlines the conributions that each of the teams in the course were able to make to their open-source project. Contributions have been categorized and are presented below:
+This chapter outlines some of the contributions made by several teams to their open-source project. Contributions have been categorized based on their types. At the end a list of all mentioned Pull Requests will be provided for more information. It should be mentioned that much more Pull Requests were filed during the course, this chapter only sketches a broad overview of the diversity.
```

Comment:
I always use open source without a dash but I see some people use it with a dash... So what's the right one? 😓